src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -60,7 +55,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     String password = request.getParameter("password");
-    userStore.addUser(username, password, /*admin=*/false);
+    userStore.addUser(username, password, false, false);
```

Comment:
Use the comments like `/*admin=*/` whenever you have plain boolean arguments so that the reader knows what they mean without having to go look up the method definition.  (Or, alternatively, use an enum so that the value itself has a meaningful name.)